client/rpc.go

```diff
@@ -70,34 +70,39 @@ func (c *Client) RPC(method string, args interface{}, reply interface{}) error {
 	}
 
 TRY:
+	var rpcErr error
+
 	server := c.servers.FindServer()
 	if server == nil {
-		return noServersErr
+		rpcErr = noServersErr
 	}
 
-	// Make the request.
-	rpcErr := c.connPool.RPC(c.Region(), server.Addr, method, args, reply)
+	if server != nil {
```

Comment:
can't this be an `else` on the previous if?

Reply:
Ah good point, thanks!